tests/routes/case-audits.test.ts

```diff
@@ -1,10 +1,11 @@
 const supertest = require('supertest');
 const Sequelize = require('sequelize');
-const app = require('../../app');
+const expressApp = require('../../app');
```

Comment:
Again, naming conflicts... it might just have been WS getting confused by the dist folder being under the source tree...